include/amici/forwardproblem.h

```diff
@@ -391,6 +395,9 @@ class ForwardProblem {
     /** current time */
     realtype t;
 
+    /** number of timepoints */
+    std::vector<realtype> timepoints;
```

Comment:
Why duplicate here and not just use `model->getTimepoint(it)`?

Reply:
Was also my first idea, but timepoints used in ForwardProblem are not necessarily the same as in `model`, as ForwardProblem does not contain those with `t = inf`. As I was not 100% sure of the behavior with respect to the code part in
https://github.com/ICB-DCM/AMICI/blob/b10a8a3514eb1ff3ed727b9c2831e45a310986f1/src/rdata.cpp#L259
I decided to go for something which is logically fully equivalent. But if there's another way to handle the lines in rdata, I'm fine to go with `model->getTimepoints`... :)

Reply:
That implementation wasn't very neat to begin. Probably makes more sense to extract maximum simulation time from forward problem and invalidate for all `model->getTimepoints` after that.

Reply:
Okay, I can change that accordingly. :+1: